python/marvin/tools/map.py

```diff
@@ -252,7 +287,7 @@ def restore(cls, path, delete=False):
 
         return marvin.core.marvin_pickle.restore(path, delete=delete)
 
-    def plot(self, array='value', xlim=None, ylim=None, zlim=None,
+    def _plot(self, array='value', xlim=None, ylim=None, zlim=None,
```

Comment:
It still works (that's how I generated the "Before" plots), but I'm fine with removing it if @albireox is.